.github/workflows/eden.yml

```diff
@@ -34,10 +34,10 @@ jobs:
       - name: run
         run: |
           ./eden config add default
-          ./eden config set default --key eve.tag --value=$(make -C eve version)
+          ./eden config set default --key eve.tag --value=$(make -s -C eve version)
           ./eden config set default --key=eve.accel --value=false
           echo > tests/workflow/testdata/eden_stop.txt
-          ./eden test ./tests/workflow
+          ./eden test ./tests/workflow -v debug || (./eden log --format json > trace.log; exit 1)
```

Comment:
A more reliable solution will be to put `./eden log` into the next step of the job. E.g.:
```
- name: Collect logs
   if: ${{ always() }}
   run: ./eden log --format json > trace.log
```